Workbooks/Workspace/Agent Health/Agent Health.workbook

```diff
@@ -0,0 +1,302 @@
+                },
+                {
+                  "durationMs": 1800000
+                },
+                {
+                  "durationMs": 3600000
+                },
+                {
+                  "durationMs": 14400000
+                },
+                {
+                  "durationMs": 43200000
+                },
+                {
+                  "durationMs": 86400000
+                },
+                {
+                  "durationMs": 172800000
+                },
+                {
+                  "durationMs": 259200000
+                },
+                {
+                  "durationMs": 604800000
+                }
+              ],
+              "allowCustom": null
+            },
+            "timeContextFromParameter": null
+          },
+          {
+            "id": "c4d8d055-6dbe-451b-bf64-2be8d9e21fd8",
+            "version": "KqlParameterItem/1.0",
+            "name": "UnhealthyCriteria",
+            "type": 2,
+            "isRequired": true,
+            "value": "5m",
+            "isHiddenWhenLocked": false,
+            "typeSettings": {
+              "additionalResourceOptions": []
+            },
+            "jsonData": "[\r\n    { \"value\":\"1m\", \"label\":\"1 minute without heartbeat\", \"selected\":false },\r\n    { \"value\":\"5m\", \"label\":\"5 minutes without heartbeat\", \"selected\":false },\r\n    { \"value\":\"30m\", \"label\":\"30 minutes without heartbeat\", \"selected\":false },\r\n    { \"value\":\"1h\", \"label\":\"1 hour without heartbeat\", \"selected\":true },\r\n    { \"value\":\"2h\", \"label\":\"2 hours without heartbeat\", \"selected\":false },\r\n    { \"value\":\"8h\", \"label\":\"8 hours without heartbeat\", \"selected\":false },\r\n    { \"value\":\"1d\", \"label\":\"1 day without heartbeat\", \"selected\":false },\r\n    { \"value\":\"2d\", \"label\":\"2 days without heartbeat\", \"selected\":false },\r\n    { \"value\":\"7d\", \"label\":\"7 days without heartbeat\", \"selected\":false }\r\n]",
+            "timeContextFromParameter": null
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.operationsmanagement/solutions"
+      },
+      "conditionalVisibility": null,
+      "name": "parameters - 1"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "datatable(State:string, Rank:int)[\"❌ Unhealthy\", 0, \"✔️ Healthy\", 1]\r\n| join kind = leftouter (Heartbeat\r\n| where TimeGenerated {TimeRange:query}\r\n| summarize LastHeartbeat = max(TimeGenerated) by Computer\r\n| extend State = iff(LastHeartbeat < ago({UnhealthyCriteria}), '❌ Unhealthy', '✔️ Healthy')\r\n| summarize Count = count() by State) on State\r\n| extend Count = iff(isempty(State1), 0, Count)\r\n| project-away State1\r\n| extend Rank = iff(State == '❌ Unhealthy' and Count == 0, 2, Rank)\r\n| order by Rank asc\r\n| extend AgentText = 'Agents'",
```

Comment:
Take out the emojis and make the icon come from thresholds

Reply:
I am fixing the formatting for tiles and thresholds. Will add icons to the template after those changes are in.